README.md

```diff
@@ -112,6 +127,20 @@ As you may notice, there are two files that need to be defined in order to confi
 
 - `file:///data/config.yml` on Linux
 - `file:c:\\data\config.yml` on Windows
+ 
+## Production deployment
+
+The example module is provided for testing purposes. Only core module should be deployed on production environment (Knot.x application as single verticle without any dependencies).
```

Comment:
Only *the* core module should be deployed *in a* production environment. (*The* Knot.x application runs as a single verticle without any dependencies).